groups/bace-iot-platform/measurements-webhook/tests.yml

```diff
@@ -0,0 +1,13 @@
+default:
+    description: Should add measurement for valid payload
+    payload: ./payloads/testPayload.json
+    expectedMeasurements:
+        - ingestionId: f1df36e6-05d9-4997-9d70-ca09d2d8695d$temperature
+          value: 28.84399414
+          date: 2023-01-01T00:00:00.000+00:00
+        - ingestionId: f1df36e6-05d9-4997-9d70-ca09d2d8695d$relative-humidity
+          value: 32.65380859
+          date: 2023-01-01T00:00:00.000+00:00
+        - ingestionId: f1df36e6-05d9-4997-9d70-ca09d2d8695d$light-intensity
+          value: 0.07
+          date: 2023-01-01T00:00:00.000+00:00
```

Comment:
Maybe add a test that shows that no measurements are created if required fields are missing?
And a newline at the end of this file 😉